### DISCUSSION TITLE
what is -r for

### DISCUSSION THREAD

**Discussion by ddst0111** [microsoft/FHIR-Converter#136] [Q&A: Liquid Engine] [answered]
just download and build the source code, when execute the command 
.\Microsoft.Health.Fhir.Liquid.Converter.Tool.exe convert  -d ./myhl7 -o ./hl72fhir 

there ask for -r --RootTemplate, and it is not mentioned in the readme.

**Comment by ranvijaykumar** [comment, marked as answer]
As mentioned in the documentation: https://github.com/microsoft/FHIR-Converter, -r stands for the root template.  It can be one of ADT_A01, OML_O21, ORU_R01, VXU_V04.

**Comment by granadacoder** [comment]
Here's a link to the source code (to enhance @ranvijaykumar 's answer)

https://github.com/microsoft/FHIR-Converter/blob/main/src/Microsoft.Health.Fhir.Liquid.Converter.Tool/Models/ConverterOptions.cs#L16L17
